src/pudl/glue/ferc1_eia.py

```diff
@@ -132,7 +132,7 @@ def get_db_plants_ferc1(
     """
     # Need to be able to use years outside the "valid" range if we're trying
     # to get new plant ID info...
-    unrecognized_years = set(years).difference(pc.WORKING_PARTITIONS['ferc1']['years'])
+    unrecognized_years = set(years).difference(Ferc1Settings.working_partitions['years'])
     if unrecognized_years:
         raise ValueError(
             f"Input years {sorted(unrecognized_years)} not available in FERC 1 DB."
```

Comment:
I'm not sure. In this context, the list of years isn't part of an ETL process. This function is used to generate the list of FERC plants to be assigned manual PUDL IDs. So `years` here is whatever list of years the user has asked to get FERC 1 Plants from and could be invalid.  However, given that we're about to access the FERC 1 DB below, it might be just as reasonable to just pull the list of all `report_year` values from the DB directly (at least for the 4 tables we're going to scan for plants) and use that as the range of years to check against -- if the user has asked for plants from years that aren't in the DB they're going to be disappointed -- and confused if those years could have hypothetically been in the DB.